src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -90,7 +91,7 @@
 import org.spigotmc.event.player.PlayerSpawnLocationEvent;
 
 import javax.imageio.ImageIO;
-import java.awt.Graphics2D;
+import java.awt.*;
```

Comment:
I often do this as well, anyhow please avoid wildcard imports. You can change this in your IDE settings in such a way that it does not autoformat to this.

Reply:
Thanks for the heads up. For some reason in this project I still had the configuration to covert the "java.awt" to a wildcard import. I've removed this and updated the PR.